### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by ZabihullahNooriWardak**
In this branch I worked on following features of the App : 
1 : webpack configed.
2 : design created.
3 : readmi added
4 : live demo link added .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,130 @@
...
+
+## 👥 Authors <a name="authors"></a>
+
+> 
+
+👤 **Zabi**
+
+- GitHub: [@githubhandle](https://github.com/ZabihullahNooriWardak)
+- Twitter: [@twitterhandle](https://twitter.com/ZabiNoo31954752)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zabi-noori-aa59a924a/)
```
Comment (Efiamotu-1) [inline README.md]: - [ ] The LinkedIn link you provided is broken, when clicked it does not show your account on the platform. Please Kindly provide the appropriate link 👍

File: README.md [hunk 2]
```diff
@@ -1,2 +1,130 @@
...
+### Install
+
+Install this project with:
+
+this project does not have need further installition just 
+you should have IDE and browser node installed and webpack too.
+
+### Usage
+
+to run this project you need Google chrome or other browser
```
Comment (Efiamotu-1) [inline README.md]: - [ ] Please update your usage to reflect this new project which requires users to use `npm start` to run the project on their browsers 👍

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @ZabihullahNooriWardak , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️Good job setting up webpack correctly
- ✔️Great job displaying the tasks
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @ZabihullahNooriWardak       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  a. 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] No linter errors
- [x] App Works as intended 💯
- [x] Nice Ui design

 

 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Pull request merged**